### PR TITLE
feat(webterminal): move webterminal from service catalog to backstage plugins

### PR DESCRIPTION
**Pull request by mimotej**
This PR moves webterminal frontend plugin with all its git history from Operate First service-catalog.

### CODE REVIEW

**Review by tumido — CHANGES_REQUESTED** [review]
As an initial transfer PR this looks neat. Please update `package.json`. 🙂

File: plugins/web-terminal/package.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@internal/backstage-plugin-web-terminal",
```
[the remark this answers is not in the document]
Comment (mimotej) [reply plugins/web-terminal/package.json]: Resolved thanks :+1:

File: plugins/web-terminal/package.json [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  "version": "0.1.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": true,
```
[the remark this answers is not in the document]
Comment (mimotej) [reply plugins/web-terminal/package.json]: Resolved

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Pull request merged**